test/testutils/syncenginetestutils.cpp

```diff
@@ -920,16 +944,15 @@ QNetworkReply *FakeQNAM::createRequest(QNetworkAccessManager::Operation op, cons
     return reply;
 }
 
-FakeFolder::FakeFolder(const FileInfo &fileTemplate, OCC::Vfs::Mode vfsMode)
```

Comment:
@TheOneRing can you verify that this constructor is doing the correct thing, including wrt. the call to `switchVfs` and the surrounding code?